react/features/notifications/middleware.js

```diff
@@ -82,30 +69,36 @@ MiddlewareRegistry.register(store => next => action => {
         return next(action);
     }
     case PARTICIPANT_UPDATED: {
-        if (typeof interfaceConfig === 'undefined' || interfaceConfig.DISABLE_FOCUS_INDICATOR) {
+        if (typeof interfaceConfig === 'undefined') {
             // Do not show the notification for mobile and also when the focus indicator is disabled.
```

Comment:
This change worries me a bit. @saghul I know you are currently looking at mobile performance, is this the setting we count on mobile for meet.jit.si and similar deployments for mobile to not spam users with many notifications.
@robertpin wouldn't this show the moderator notification no matter what? This means on meet.jit.si we will see it for every participant that joins a meeting?

Reply:
Aaa this is only for the local participant oh :)

Reply:
Right now this setting is only used to disable the moderator notification and the moderator badge on the video thumbnail.
The answer to the question is no, the code was modified so now only the user that is granted moderation rights will see a notification (something like "You're now a moderator"), and the others won't see anything